pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -85,6 +85,7 @@ public abstract class BaseBrokerRequestHandler implements BrokerRequestHandler {
   protected final Set<String> _trackedHeaders;
   protected final BrokerRequestIdGenerator _requestIdGenerator;
   protected final long _brokerTimeoutMs;
+  protected final long _brokerPassiveTimeoutMs;
```

Comment:
If this is only meant for use with MSE, should this be in `MultiStageBrokerRequestHandler` instead?

Reply:
SSE could use it, but let's move it to the MSE handler for the time being.